packages/react/src/SelectPanel/SelectPanel.tsx

```diff
@@ -76,11 +76,11 @@ interface SelectPanelBaseProps {
     open: boolean,
     gesture: 'anchor-click' | 'anchor-key-press' | 'click-outside' | 'escape' | 'selection' | 'cancel',
   ) => void
+  secondaryAction?: React.ReactElement
```

Comment:
does this mean consumers could technically put whatever they want in here? if so we should run this by @emilybrick

Reply:
It is, I thought of making it more closed but then we might have issues with links, loading states etc.

Reply:
I think the secondary action should be limited to buttons and links 🤔 - not anything. That said, I don't _think_ people will abuse this (if we keep it open to any custom React element), but I don't want to assume.